bpf/lib/nodeport.h

```diff
@@ -2661,6 +2662,8 @@ int tail_nodeport_nat_egress_ipv4(struct __ctx_buff *ctx)
 	struct iphdr *ip4;
 	__s8 ext_err = 0;
 #ifdef TUNNEL_MODE
+	__u8 cluster_id __maybe_unused = (__u8)ctx_load_meta(ctx, CB_CLUSTER_ID_EGRESS);
```

Comment:
The `__maybe_unused` isn't needed, correct?

Reply:
Ah, otherwise we still pass it to `lookup_ip4_remote_endpoint()` but then no-one uses the result :+1: .